sys/include/net/gnrc/tcp/config.h

```diff
@@ -26,18 +26,43 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup net_gnrc_tcp_conf GNRC TCP compile configurations
+ * @ingroup net_gnrc_conf
+ *
+ * ## Calculating RTO
+ * To calculate retransmission timeout (RTO), Round Trip Time (RTT) needs to be
+ * taken into account. SRTT (smoothed round-trip time) and RTTVAR (round-trip
+ * time variation) are hence calculated as follows:
+ *
+ *     RTTVAR <- (1 - beta) * RTTVAR + beta * |SRTT - R'|
+ *     SRTT <- (1 - alpha) * SRTT + alpha * R'
+ *
+ * where alpha ( @ref CONFIG_GNRC_TCP_RTO_A_DIV ) and beta
+ * ( @ref CONFIG_GNRC_TCP_RTO_B_DIV) are constants, and R' is the instantaneous
+ * RTT value.
+ *
```

Comment:
The comment above is a little bit misleading. CONFIG_GNRC_TCP_RTO_A_DIV is not the alpha value, it is the denominator for alpha.

The default value for Alpha is 1/8, therefore  CONFIG_GNRC_TCP_RTO_A_DIV is 8. The same goes for beta and CONFIG_GNRC_TCP_RTO_B_DIV.